phpstan.neon

```diff
@@ -1,6 +1,6 @@
 parameters:
     parallel:
-        maximumNumberOfProcesses: 2
+        maximumNumberOfProcesses: 8
```

Comment:
I am not sure we want to change this value (I suppose it is set to a low value for a reason, maybe for the CI ?).
When I had the same need for more cores, I did this : https://github.com/glpi-project/glpi/pull/17420.

However it was done on main, I probably should have done it in bugfixes instead

Reply:
To be clear I am 100% OK for the change is there was no specific reasons for the low cores count, I just don't know why it is this way.

Reply:
BE carefull with taht change, see https://github.com/glpi-project/glpi/pull/12689 (maybe it's no longer relevant, I do not use docker myself).

Reply:
Oh no, I didn't want to commit it. It was to test only on my workstation if it was faster, but the difference isn't huge.